app/assets/stylesheets/modules/_details.scss

```diff
@@ -0,0 +1,128 @@
+.details-hero {
+  background-color: $white;
+  text-align: center;
+
+  &-title {
+    margin-bottom: em(5);
+  }
+
+  &-compliment {
+    display: inline-block;
+
+    &-refresh {
+      background: image-url("refresh.png");
+      background-repeat: no-repeat;
+      background-size: 14px;
+      display: inline-block;
+      height: em(15);
+      width: em(15);
+      margin-left: em(10);
+
+      &:hover {
+        opacity: 0.5;
+        cursor: pointer;
+      }
+    }
+  }
+
+  &-link {
+    font-size: $base-font-size/1.3;
+    font-weight: 300;
+    font-style: italic;
+    margin-bottom: $secondary-spacing;
+  }
+
+  &-text {
+    @include span-columns (6 of 12);
+    @include shift(3);
+
+    p {
+      margin-bottom: em(30);
+    }
+  }
+
+  &-yes {
+    color: $base-accent-color;
+    font-weight: 600;
+    letter-spacing: 1.3px;
+
+    &-arrow {
+      width: 0; 
+      height: 0; 
+      border-left: 7px solid transparent;
+      border-right: 7px solid transparent;
+      border-top: 7px solid $base-accent-color;
+      font-size: 0;
+      line-height: 0;
+    }
+  }
+
+
+}
+
+.details-form {
+  color: $white;
+
+  &-title {
+    text-align: center;
+  }
+
+  &-subtitle {
+    text-align: center;
+  }
+
+  &-field {
+    @include span-columns(6 of 12);
+    @include shift(3);
+
+    &-half {
+      display: inline-block;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets